src/nvim/os/unix_defs.h

```diff
@@ -31,12 +31,6 @@
 #ifndef USR_EXRC_FILE
 # define USR_EXRC_FILE "~/.exrc"
 #endif
-#ifndef USR_VIMRC_FILE
-# define USR_VIMRC_FILE "~/.nvimrc"
```

Comment:
@fmoralesc I’m always a bit surprised when people bring up Mac OS X GUI level configuration locations in the context of the XDG spec. When it comes to the command line, OS X is a BSD (OK, a user space BSD layer wrapped around a Mach micro kernel, but that is academic doe the purpose of this thread) and as such firmly rooted in the \*nix world. The relevant Vim OS class is `macunix`, BTW, which says it pretty clearly ;).

To my understanding, this means the XDG default locations can and should match the spec: `~/.config`, `~/.cache` etc., an understanding shared by all XDG aware CLI applications I have met on on OS X (front and centre: Git).